src/gui/render/opengl_renderer.cpp

```diff
@@ -386,7 +388,6 @@ void OpenGlRenderer::PresentFrame()
 {
 	glClear(GL_COLOR_BUFFER_BIT);
 
-	frame_count++;
```

Comment:
Explanation for this change? Just want to make sure this isn't a mistake that snuk in as part of a large commit.